Formula/orgalorg.rb

```diff
@@ -0,0 +1,22 @@
+class Orgalorg < Formula
+  desc "Parallel SSH commands executioner and file synchronization tool"
+  homepage "https://github.com/reconquest/orgalorg"
+  url "https://github.com/reconquest/orgalorg.git",
+      tag:      "1.0.1",
+      revision: "7ccd02c521f79e04720aa929e21715d46668162f"
+  license "MIT"
+  head "https://github.com/reconquest/orgalorg.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-mod=mod", "-ldflags", "-s -w -X main.version=#{version}", *std_go_args
+  end
+
+  test do
+    assert_match "orgalorg - files synchronization on many hosts.",
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
testing of this tool involves remote hosts, I don't have idea how to test it
tool is used for working with remote hosts and not suppose to be used for localhost only

Reply:
So connect to a remote host?

Reply:
Could you please suggest any examples for such tests, I looked through all the core formulas and there are a lot of version and help checks, so I didn't found any remote host checks or test.

Reply:
Just run something a user would and assert that it outputs as expected.

Reply:
Is that enough for now?